lendingbot.py

```diff
@@ -15,187 +15,87 @@
-        sleep_time_inactive = sleep_time
-        print "!!! Please update to new config that includes Inactive Mode. !!!"  # Update alert.
-    api_key = config.get("API", "apikey")
-    api_secret = config.get("API", "secret")
-    min_daily_rate = Decimal(config.get("BOT", "mindailyrate")) / 100
-    max_daily_rate = Decimal(config.get("BOT", "maxdailyrate")) / 100
-    spread_lend = int(config.get("BOT", "spreadlend"))
-    gap_bottom = Decimal(config.get("BOT", "gapbottom"))
-    gap_top = Decimal(config.get("BOT", "gaptop"))
-    if config.has_option("BOT", "sixtyDayThreshold"):
-        sixty_day_threshold = float(config.get("BOT", "sixtydaythreshold")) / 100
-        xday_threshold = sixty_day_threshold
-        xdays = "60"
-    else:
-        xday_threshold = Decimal(config.get("BOT", "xdaythreshold")) / 100
-        xdays = str(config.get("BOT", "xdays"))
-    auto_renew = int(config.get("BOT", "autorenew"))
-    if config.has_option('BOT', 'outputCurrency'):
-        output_currency = config.get('BOT', 'outputCurrency')
-    if config.has_option('BOT', 'maxtolent'):
-        maxtolent = Decimal(config.get('BOT', 'maxtolent'))
-    if config.has_option('BOT', 'maxpercenttolent'):
-        maxpercenttolent = Decimal(config.get('BOT', 'maxpercenttolent')) / 100
-    if config.has_option('BOT', 'maxtolentrate'):
-        maxtolentrate = Decimal(config.get('BOT', 'maxtolentrate')) / 100
-    if config.has_option('BOT', 'minloansize'):
-        min_loan_size = Decimal(config.get("BOT", 'minloansize'))
-    if config.has_option('BOT', "transferableCurrencies"):
-        transferable_currencies = (config.get("BOT", "transferableCurrencies")).split(",")
-    else:
-        transferable_currencies = []
-
-    if config.has_option("BOT", "coinconfig"):
-        try:
-            # parsed
-            coinconfig = (json.loads(config.get("BOT", "coinconfig")))
-            for cur in coinconfig:
-                cur = cur.split(':')
-                coincfg[cur[0]] = dict(minrate=(Decimal(cur[1])) / 100, maxactive=Decimal(cur[2]),
-                                       maxtolent=Decimal(cur[3]), maxpercenttolent=(Decimal(cur[4])) / 100,
-                                       maxtolentrate=(Decimal(cur[5])) / 100)
-        except Exception as e:
-            print "Coinconfig parsed incorrectly, please refer to the documentation. Error: " + e
-            pass
+    transferable_currencies = []
+coincfg = {}
+if config.has_option("BOT", "coinconfig"):
+    try:
+        # parsed
+        coinconfig = (json.loads(config.get("BOT", "coinconfig")))
+        for cur in coinconfig:
+            cur = cur.split(':')
+            coincfg[cur[0]] = dict(minrate=(Decimal(cur[1])) / 100, maxactive=Decimal(cur[2]),
+                                   maxtolent=Decimal(cur[3]), maxpercenttolent=(Decimal(cur[4])) / 100,
+                                   maxtolentrate=(Decimal(cur[5])) / 100)
+    except Exception as e:
```

Comment:
we can consider reducing min variable length to 1 (or renaming Exceptions to ex)
To reduce min variable name to 1:
[a-z_][a-z0-9_]{2,30}$ -> [a-z_][a-z0-9_]{*1*,30}$